mbed_lib.json

```diff
@@ -1,6 +1,9 @@
 {
     "name": "ism43362",
     "config": {
+	    "wifi-country-code": {
+            "help": "Country Code to specify channel number"
```

Comment:
Can you explain where to find the list of supported countries and how they are coded (where did you get tables from ? )

Reply:
You can find the information at the following address:
https://www.juniper.net/documentation/en_US/release-independent/junos/topics/reference/specifications/access-point-ax411-country-channel-support.html